src/appshell/view/appmenumodel.cpp

```diff
@@ -372,13 +372,16 @@ MenuItem* AppMenuModel::makeDiagnosticMenu()
 #endif
 
     if (globalConfiguration()->devModeEnabled()) {
+#ifndef NDEBUG
```

Comment:
Why limiting all these options to debug builds? They work just fine in release builds (at least, for me). The corrupted measures option was the only one that was still limited to debug mode, as far as I know.

Reply:
None of them shows in a non-debug build, unless you do what I described in https://github.com/musescore/MuseScore/issues/24998#issuecomment-2386399458

Reply:
But I can certainly enable them

Reply:
> None of them shows in a non-debug build, unless you do what I described in [#24998 (comment)](https://github.com/musescore/MuseScore/issues/24998#issuecomment-2386399458)

That's because of the `if (globalConfiguration()->devModeEnabled())`. I don't understand what benefit adding `#ifndef NDEBUG` would bring: that would mean that they are not available in release builds _even_ after enabling dev mode. 

(by the way, enabling dev mode is done by by opening the About dialog, then clicking the app icon there three times, and restarting the program. Note that there is no visual feedback after clicking the app icon thrice, until you restart.)

Reply:
Oh, oh, an Easter egg ;-)
As said: I'm fine with enabling them, will do shortly

Reply:
Done